controllers/machineset_controller.go

```diff
@@ -126,21 +127,41 @@ func (r *ReconcileMachineSet) ProcessMachineSet(ctx context.Context, machineSet
 		// Update labels in machine
 		err = r.updateLabelsInMachine(ctx, machine, expectedLabels)
 		if err != nil {
```

Comment:
I think we should emit a metric when we succeed as well. This would give us a better picture of how often customers use this feature (business likes knowing that and we should too) as well as respond to problems.